brage-import/src/main/java/no/sikt/nva/brage/migration/lambda/BrageEntryEventConsumer.java

```diff
@@ -100,43 +100,51 @@ public BrageEntryEventConsumer() {
     }
 
     @Override
-    public Publication handleRequest(S3Event s3Event, Context context) {
+    public PublicationRepresentation handleRequest(S3Event s3Event, Context context) {
+        resetRuntime();
```

Comment:
Added a resetRuntime as I noticed we are setting som variables, and later using them for error reports. Theoretically if two posts fails during a lambda lifetime, the second report might get generated in part from the previous fail.